Sources/PackageModel/UserToolchain.swift

```diff
@@ -113,6 +116,36 @@ public final class UserToolchain: Toolchain {
 
     // MARK: - public API
 
+    public static func determineLibrarian(triple: Triple, binDir: AbsolutePath,
+                                          useXcrun: Bool,
+                                          environment: EnvironmentVariables,
+                                          searchPaths: [AbsolutePath]) throws
+            -> AbsolutePath {
+        let variable: String = triple.isDarwin() ? "LIBTOOL" : "AR"
+        let tool: String = {
+            if triple.isDarwin() { return "libtool" }
+            if triple.isWindows() {
+                // TODO(5719) cascade differnt librarians based upon flags and
+                // environment variables
+                return "lld-link"
```

Comment:
Do you mind adding `llvm-ar` as the archiver for Android here and in the test files below? The Android NDK only uses llvm-ar now, which is why [I've been patching llbuild to use llvm-ar instead](https://github.com/termux/termux-packages/blob/master/packages/swift/llbuild-lib-BuildSystem-BuildSystem.cpp.patch).

Reply:
Oh, didn't realize that Android had switched to `llvm-ar`, should be easy enough to add that.  Thanks for catching that.

Reply:
Thanks for adding Android, looks good.